AdaptiveUiCodelab/app/src/main/java/com/example/reply/ui/ReplyListContent.kt

```diff
@@ -92,15 +89,23 @@ fun ReplyListAndDetailContent(
 @Composable
 fun ReplyEmailListItem(
     email: Email,
-    modifier: Modifier = Modifier.padding(horizontal = 16.dp, vertical = 4.dp)) {
-    Card(modifier = modifier) {
+    onEmailClick: (Email) -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Card(
+        modifier = modifier
+            .padding(horizontal = 16.dp, vertical = 4.dp)
+            .clickable {
+                onEmailClick(email)
+            }
```

Comment:
Nit: There's an overload of `Card` with an `onClick` that we should probably use instead here

Reply:
Done